banking-app/scripts/setup_governance.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+set -euo pipefail
+
+declare enclave_type=""
+
+function usage {
+    echo ""
+    echo "Setup the CCF network."
+    echo ""
+    echo "usage: ./setup_governance.sh --serverIP <IPADDRESS>"
+    echo ""
+    echo "  --serverIP  string      The IP address of the primary CCF node"
+    echo ""
+    exit 0
+}
+
+function failed {
+    printf "Script failed: %s\n\n" "$1"
+    exit 1
+}
+
+# parse parameters
+
+if [[ $# -lt 2 || $# -gt 2 ]]; then
+    usage
+    exit 1
+fi
+
```

Comment:
I haven't tested this, but looks like the `--help` switch (which is used alone) will not work if you have this validation.
The final result will be the same, as `--help` outputs the usage function, so in the way it's written, the output is correct, however not through the correct flow. If I'm not mistaken we may be adding a trap here :)

You can leave the checks to the for loop code below, and the correct parameter check will be made on lines 41-45

Reply:
Great spot - I have changed this to always check greater than the maximum number of parameters so that help works.  If you are happy, can you resolve.